### PR TITLE
Add initial content for splinter-docs repo

### PR DESCRIPTION
**Pull request by chenette**
- Add README content, LICENSE, and MAINTAINERS.md

- Add doc subdirectories and example template files. Also add initial FAQ entries and glossary terms.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+| Dave Cecchi | davececchi |
+| Eloá Franca Verona | eloaverona |
+| James Mitchell | jsmitchell |
+| Logan Seeley | ltseeley |
+| Peter Schwarz | peterschwarz |
+| Ryan Banks | RyanLassigBanks |
+| Ryan Beck-Buysse | rbuysse |
+| Shannyn Telander | shannynalayna |
+| Shawn Amundson | vaporos |
+| Jason Walker | desktophero |
```
Comment (rbuysse) [inline MAINTAINERS.md]: Can you alphabetize this?

Comment (chenette) [reply MAINTAINERS.md]: Fixed: Alphabetized by first name.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -15,6 +15,4 @@
 | Ryan Beck-Buysse | rbuysse |
 | Shannyn Telander | shannynalayna |
 | Shawn Amundson | vaporos |
-| Jason Walker | desktophero |
```
Comment (agunde406) [inline MAINTAINERS.md]: This removal should be in the first commit, not this one.

Comment (chenette) [reply MAINTAINERS.md]: Fixed.

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@agunde406 @chenette @davececchi @dplumb94 @eloaverona @jsmitchell @ltseeley @peterschwarz @rbuysse @RyanLassigBanks @shannynalayna @vaporos @zapplebee
```
Comment (agunde406) [inline CODEOWNERS]: This is not the write format, it needs "*       " in front of the first name

Comment (chenette) [reply CODEOWNERS]: Fixed.

**Pull request merged**